Misc/NEWS.d/next/Library/2018-09-14-10-38-18.bpo-31177.Sv91TN.rst

```diff
@@ -0,0 +1 @@
+Skip deleted attributes while calling :meth:`mock.reset_mock`.
```

Comment:
I'd suggest rewording this to something around:

"Fix bug that prevented using `reset_mock` on mock instances with deleted attributes"